test/functional/wallet_listtransactions.py

```diff
@@ -277,6 +278,34 @@ def normalize_list(txs):
         assert_equal(['pizza2'], self.nodes[0].getaddressinfo(addr2)['labels'])
         assert_equal(['pizza3'], self.nodes[0].getaddressinfo(addr3)['labels'])
 
+    def run_coinjoin_test(self):
+        self.log.info('Check "coin-join" transaction')
+        input_0 = next(i for i in self.nodes[0].listunspent(query_options={"minimumAmount": 0.2}, include_unsafe=False))
+        input_1 = next(i for i in self.nodes[1].listunspent(query_options={"minimumAmount": 0.2}, include_unsafe=False))
+        raw_hex = self.nodes[0].createrawtransaction(
+            inputs=[
+                {
+                    "txid": input_0["txid"],
+                    "vout": input_0["vout"],
+                },
+                {
+                    "txid": input_1["txid"],
+                    "vout": input_1["vout"],
+                },
+            ],
+            outputs={
+                self.nodes[0].getnewaddress(): 0.123,
+                self.nodes[1].getnewaddress(): 0.123,
+            },
+        )
+        raw_hex = self.nodes[0].signrawtransactionwithwallet(raw_hex)["hex"]
+        raw_hex = self.nodes[1].signrawtransactionwithwallet(raw_hex)["hex"]
+        txid_join = self.nodes[0].sendrawtransaction(hexstring=raw_hex, maxfeerate=0)
+        fee_join = self.nodes[0].getmempoolentry(txid_join)["fees"]["base"]
+        # Fee should be correct: assert_equal(fee_join, self.nodes[0].gettransaction(txid_join)['fee'])
+        # But it is not, see for example https://github.com/bitcoin/bitcoin/issues/14136:
+        assert fee_join != self.nodes[0].gettransaction(txid_join)["fee"]
```

Comment:
As this tests confirms that a bug still exists for the `gettransaction` rpc call, would it make sense to further document the issue by adding another assertion from `nodes[1]` perspective? Because in the case that inputs are not the same, the resulting fee from `gettransaction` is different based on which side called the RPC.

Reply:
I think the test as-is now is sufficient